Towny/src/main/java/com/palmergames/bukkit/towny/utils/CombatUtil.java

```diff
@@ -147,12 +147,18 @@ private static boolean preventDamageCall(TownyWorld world, Entity attackingEntit
 				 * Both townblocks are not Arena plots.
 				 */
 				if (!isArenaPlot(attackerTB, defenderTB)) {
+
 					/*
-					 * Check if we are preventing friendly fire between allies
-					 * Check the attackers TownBlock and it's Town for their PvP status, else the world.
-					 * Check the defenders TownBlock and it's Town for their PvP status, else the world.
+					 * Player does not have the towny.admin permission.
 					 */
-					cancelled = preventFriendlyFire(attackingPlayer, defendingPlayer, world) || preventPvP(world, attackerTB) || preventPvP(world, defenderTB) || preventJailedPVP(defendingPlayer, attackingPlayer);
+					if (!attackingPlayer.hasPermission("towny.admin")) {
```

Comment:
```suggestion
					if (!TownyUniverse.getInstance().getPermissionSource().isTownyAdmin(attackingPlayer)) {
```
This should go through the permission source so that admins using adminbypass mode are treated like normal players.